File: Ejercicios Propuestos 1/Ejercicio 20/EJERCICIO_20.cpp
#include <iostream>

using namespace std;

int main() {
    int posicion_n, contador, num_triangular;
    
    cout << "Ingrese la posicion del numero triangular (valor de n): ";
    cin >> posicion_n;
    
    if (posicion_n > 0) {
        num_triangular = 0;
        for (contador = 1; contador <= posicion_n; contador++) {
            num_triangular = num_triangular + contador;
        }
        cout << "El numero triangular en la posicion " << posicion_n << " es: " << num_triangular << endl;
    } else {
        cout << "Error: Debe ingresar una posicion valida (entero positivo)." << endl;
    }
    
    return 0;
}

File: Ejercicios Propuestos 1/Ejercicio 20/Ejercicio 20.jl
print("Ingrese la posicion del numero triangular (valor de n): ")
posicion_n = parse(Int, readline())

if posicion_n > 0
    num_triangular = 0
    for contador in 1:posicion_n
        global num_triangular += contador
    end
    println("El número triangular en la posicion ", posicion_n, " es: ", num_triangular)
else
    println("Error: Debe ingresar una posicion valida (entero positivo).")
end
